db/migrate/20210818065350_add_picture_to_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddPictureToItems < ActiveRecord::Migration[6.1]
+  def change
+    add_column :items, :picture, :string
```

Comment:
read about active storage and use it for pics

Reply:
Alright sir